pkg/services/ngalert/ngalert.go

```diff
@@ -519,6 +521,10 @@ func (ng *AlertNG) init() error {
 	return DeclareFixedRoles(ng.AccesscontrolService, ng.FeatureToggles)
 }
 
+func useSequences(featureToggles featuremgmt.FeatureToggles, jitterStrategy schedule.JitterStrategy) bool {
```

Comment:
Do we actually want to make this a global flag? I was under the impression that we would only support this for converted DMA rules for now and that we have a way to identify those rules. All or nothing will make rolling this out in mixed environments a bit more precarious I think.